src/sentry/integrations/pagerduty/client.py

```diff
@@ -47,6 +47,7 @@ def send_trigger(
             custom_details = serialize(data, None, ExternalEventSerializer())
             summary = custom_details["message"][:1024] or custom_details["title"]
             link_params = {"referrer": "pagerduty_integration"}
+            client_url = group.get_absolute_url(params=link_params)
```

Comment:
Could you move this below:
``` python
            if notification_uuid:
                link_params["notification_uuid"] = notification_uuid
```
The test is failing b/c we're missing this notification uuid

Reply:
@schew2381 Thank you. Updated